### PR TITLE
<Rituraj>Push Notification in Alert MS.

### PR DESCRIPTION
**Pull request by RiturajAzad-RA**
Added code for push notification in Alert MS

### CODE REVIEW

File: JCB-Alerts/src/main/java/com/wipro/jcb/livelink/app/alerts/controller/AlertController.java [hunk 1]
```diff
@@ -408,4 +425,153 @@ public ResponseEntity<?> removeNotification(@RequestHeader(MessagesList.LoggedIn
...
+            List<String> users = osType.equalsIgnoreCase("Android")
+                    ? userNotificationDetailRepo.getAndroidFCMKeyForSalesforceUniqueUser(userName, userDetails)
+                    : userNotificationDetailRepo.getIosFCMKeyForSalesforce(userName, userDetails);
+
+
+            switch (type) {
+                case "Alert" -> {
+                    log.info("Alert Push Notification From Firebase - Start");
+                    MessageContent pushNotification = new MessageContent();
+                    pushNotification.setData(new Data("Alert Description", "Alert Event Type", "Alert", "New Alert Event Type Alert"));
```
Comment (GokulAher) [inline JCB-Alerts/src/main/java/com/wipro/jcb/livelink/app/alerts/controller/AlertController.java:480]: looks like massage is hardcoded in the java ? can we move it to the CONSTANTS file?

**Review by GokulAher — COMMENTED** [review]
pl. incorporate the code review comments

File: JCB-Alerts/src/main/java/com/wipro/jcb/livelink/app/alerts/controller/AlertController.java [hunk 2]
```diff
@@ -408,4 +425,153 @@ public ResponseEntity<?> removeNotification(@RequestHeader(MessagesList.LoggedIn
...
+                        alertUtilities.sendNotificationUsingV1(pushNotification);
+                    }
+
+                    log.info("Breakfast Push Notification From Firebase - End");
+                }
+                case "Salesforce" -> {
+                    log.info("notification list :{}", users.size());
+                    log.info("Salesforce Push Notification From Firebase - Start");
+                    MessageContent pushNotification = new MessageContent();
+                    pushNotification.setData(new Data("Please help us in reaching our goal of 100 % fantastic sales experience by answering a few questions about your recent purchase.",
```
[the remark this answers is not in the document]
Comment (RiturajAzad-RA) [reply JCB-Alerts/src/main/java/com/wipro/jcb/livelink/app/alerts/controller/AlertController.java:511]: incorporated the changes in a seperate constant file

File: JCB-Alerts/src/main/java/com/wipro/jcb/livelink/app/alerts/controller/AlertController.java (hunk 1 above)
Comment (RiturajAzad-RA) [reply JCB-Alerts/src/main/java/com/wipro/jcb/livelink/app/alerts/controller/AlertController.java:480]: incorporated the changes in a separate constant file

**Review by GokulAher — APPROVED** [review]
Changes are incorporated, and the code looks ok to me

**Pull request merged**